gslib/commands/perfdiag.py

```diff
@@ -235,13 +250,18 @@
 
 
 <B>NOTE</B>
-  The perfdiag command collects system information. It collects your IP address,
-  executes DNS queries to Google servers and collects the results, collects
-  network statistics information from the output of netstat -s, and looks at the
-  BIOS product name string. It will also attempt to connect to your proxy server
-  if you have one configured and will look up the location and storage class of
-  the bucket being used for performance testing. None of this information will
-  be sent to Google unless you choose to send it.
+  The ``perfdiag`` command collects system information, such as the following
+  items: 
+  
+  + the requester's IP address
+  + executes DNS queries to Google servers and collects the results
+  + network statistics information from the output of ``netstat -s`` and
+    evaluates the BIOS product name string
+  + If configured, attempts to connect to the requester's proxy server
+    to look up the location and storage class of the bucket being used for
+    performance testing
+  
+  None of this information will be sent to Google unless you choose to send it.
```

Comment:
How would the user approve or disapprove of sending the data?

Reply:
@dilipped Would you be able to help answer this? Thanks!

Reply:
I think it does not send the data automatically. You have to send it manually if you wish to. See https://cloud.google.com/storage/docs/gsutil/commands/perfdiag#providing-diagnostic-output-to-cloud-storage-team